uaclient/entitlements/fips.py

```diff
@@ -325,6 +325,10 @@ def install_all_available_fips_upgrades(
             )
         ]
 
+        # See GH #3390
+        if system.get_release_info().series == "jammy":
```

Comment:
Any reason why we are not creating a jammy entry under `FIPS_CONTAINER_CONDITIONAL_PACKAGES` ?